ngraph/python/src/ngraph/utils/node_factory.py

```diff
@@ -8,7 +8,7 @@
 
 from ngraph.impl import Node, Output
 
-DEFAULT_OPSET = "opset6"
+DEFAULT_OPSET = "opset8"
```

Comment:
Also this change should be also in separate PR.

Reply:
done